### PR TITLE
Adding setup-avahi-discovery role

### PR DESCRIPTION
**Pull request by chbrown13**
Ansible role for the automation-tools setup_avahi_discovery task

### CODE REVIEW

File: roles/avahi/tasks/restart_avahi.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+- name: "Restart dbus"
+  service:
+    name: dbus
+    state: restarted
```
Comment (JacobCallahan) [inline roles/avahi/tasks/restart_avahi.yml:5]: +1

File: roles/avahi/tasks/add_mdns_service.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+
```
Comment (ehelms) [inline roles/avahi/tasks/add_mdns_service.yml]: Can remove

**Review by ehelms — APPROVED** [review]

**Comment by kbidarkar** [comment]
@chbrown13, Can you please add this to the excel sheet, which was shared.

**Review by kbidarkar — CHANGES_REQUESTED** [review]
Ansible stuff appears to be fine, but request to please squash these commits.

I would still say the same even though Github has the option to squash the commits and merge them.

As it would probably pick the commit message related to docker roles instead of setup-avahi roles.

**Comment by pgagne** [comment]
Why do we need Avahi again?

**Comment by kbidarkar** [comment]
when I saw this PR, I thought why we need Avahi with SatLab, but on thinking again realized,

Probably we want to have this repo more generic and not keeping SatLab focused and hence feel Avahi is ok to have. 
If we go down the road of having only those stuff which would be required with SatLab, we would end up not being able to share the repo with anyone else outside our team, who do not have access to SatLab. Which we may not want to do.

Though, currently, I would not include avahi roles into the playbooks created already, but probably have another playbook.

**Comment by kbidarkar** [comment]
Thanks, @chbrown13   for squashing the commits.

**Review by kbidarkar — APPROVED** [review]

File: roles/avahi/tasks/restart_avahi.yml (hunk 1 above)
Comment (lpramuk) [reply roles/avahi/tasks/restart_avahi.yml:5]: -1

**Pull request merged**